std/uni.d

```diff
@@ -7816,7 +7821,7 @@ if (isInputRange!S1 && isSomeChar!(ElementEncodingType!S1)
         // one of chars is not cased at all
         return diff;
     }
-    return str2.empty ? 0 : -1;
```

Comment:
if r2.empty it should return 0 not -1.
a proper check is
```d
 int(r2.empty) - 1;
```

Reply:
Pushed a fix.